pkg/proc/disasm_arm64_arch.go

```diff
@@ -0,0 +1,212 @@
+// +build linux,arm64
+
+package proc
+
+import (
+	"golang.org/x/arch/arm64/arm64asm"
+)
+
+var maxInstructionLength uint64 = 15
+
+type archInst struct {
+	Inst arm64asm.Inst
+	Len  int
```

Comment:
Do we need this? `asmDecode` always puts length at 4, which as I understand it is the instruction length in ARM state, as opposed to 2 in thumb state. Can we always assume ARM state (and thus remove this field and just return 4 in `(inst *archInst) Size()`? If not, is there a way to infer from `arch.Instr` the size or at least which mode we're in?

Reply:
I confirmed that there is no thumb code in arm64, it only exists in arm arch. So, I deleted len arg. Please review.